### PR TITLE
fix `bun create` test `should create template from local folder`

### PR DESCRIPTION
**Pull request by dylan-conway**
### What does this PR do?
Directory should have `.iterate = true`.

Adds code for copying local folders on windows.

### How did you verify your code works?
manually and updated test

### CODE REVIEW

File: src/cli/create_command.zig [hunk 1]
```diff
@@ -456,7 +456,7 @@ pub const CreateCommand = struct {
                 node.name = "Copying files";
                 progress.refresh();
 
-                const template_dir = std.fs.openDirAbsolute(filesystem.abs(&template_parts), .{}) catch |err| {
```
Comment (Jarred-Sumner) [inline src/cli/create_command.zig:459]: I don't think `.iterate` does anything on macOS

I think this is unlikely to fix the issue

File: src/cli/create_command.zig [hunk 2]
```diff
@@ -271,8 +271,8 @@ pub const CreateCommand = struct {
         const destination = try filesystem.dirname_store.append([]const u8, resolve_path.joinAbs(filesystem.top_level_dir, .auto, dirname));
 
         var progress = std.Progress{};
-        var node = progress.start(try ProgressBuf.print("Loading {s}", .{template}), 0);
```
Comment (dylan-conway) [inline src/cli/create_command.zig:274]: might need to do this everywhere, not sure yet

File: src/cli/create_command.zig (hunk 1 above)
Comment (dylan-conway) [reply src/cli/create_command.zig:459]: > I don't think `.iterate` does anything on macOS
> 
> I think this is unlikely to fix the issue

You can repro this issue on linux and windows, but it does not fix the flakiness on macos

Comment (paperclover) [reply src/cli/create_command.zig:459]: the directory is iterated. iterate should be passed

**Pull request merged**